tokentrim/tokentrim.py

```diff
@@ -13,12 +13,9 @@
   'code-llama': 1048, # I'm not sure this is correct.
 }
 
-def num_tokens_from_messages(messages: List[Dict[str, Any]],
-                             model) -> int:
-  """
-  Function to return the number of tokens used by a list of messages.
-  """
+MAX_ITERATIONS = 12
```

Comment:
No idea if this is the right amount of iterations or not. I just chose a number that seemed reasonable, but like we had probably attempted to trim too many times if we were still going.